examples/dgmulti_1d/elixir_euler_flux_diff.jl

```diff
@@ -11,9 +11,7 @@ equations = CompressibleEulerEquations1D(1.4)
 initial_condition = initial_condition_convergence_test
 source_terms = source_terms_convergence_test
 
-cells_per_dimension = (8, )
-vertex_coordinates, EToV = StartUpDG.uniform_mesh(dg.basis.elementType, cells_per_dimension...)
-mesh = VertexMappedMesh(vertex_coordinates, EToV, dg, is_periodic=(true,))
+mesh = CartesianMesh(dg, cells_per_dimension=(8,), is_periodic=(true,))
```

Comment:
While we're already talking about name changes and deprecations: Would it make sense to unify the argument names indicating periodic meshes? Currently, we use `periodic` for our Kopriva-style DGSEM meshes and `is_periodic` for our Hesthaven-Warburton-style `DGMulti` meshes.

Reply:
I am all for consistency. IIRC, we had a discussion about the naming here before, but I don't remember any details. I think back then it was between "periodic" and "periodicity". I don't have a strong opinion here, but would tend to favor the status quo (="periodic") if just for avoiding additional work.

Reply:
IIRC, ishould be `periodicity` to math the other mesh types - please check and deprecate the old keyword.

Reply:
Did you also deprecate the old keyword? You can do this as in
https://github.com/ranocha/SummationByPartsOperators.jl/blob/659ccd22ddd54964c70ad3648d384f3f2324384f/src/periodic_operators.jl#L637-L639 
for example

Reply:
@ranocha I don't understand how `Base.depwarn` would work in this case. In https://github.com/ranocha/SummationByPartsOperators.jl/blob/659ccd22ddd54964c70ad3648d384f3f2324384f/src/periodic_operators.jl#L637-L639, you seem to have different argument types for `mode`, but here the argument type stays the same, only the name changes.

Reply:
Nvm, this seems to work https://discourse.julialang.org/t/deprecate-keyword-argument/33740.

Reply:
OK, I think I got it working in https://github.com/trixi-framework/Trixi.jl/pull/1012/commits/e690f7d49c359bc0f8c234ed89f77e1e47d7d2c7 now.